README.md

```
@@ -177,6 +184,68 @@ When clicking on it, it will open a new window with the hot reload status.
 ![](assets/compose-hot-reload-console.png)
 </details>
 
+## 🍎 Build for iOS target
+
+The support of iOS works more or less _as-is_ and gets the job done. It's provided without guarantees, use at your own risk.  
```

markdownlint-cli2 (0.17.2)

Emphasis style
Expected: asterisk; Actual: underscore

(MD049, emphasis-style)